dask/dataframe/core.py

```diff
@@ -1462,6 +1462,18 @@ def append(self, other):
             other = other.to_frame().T
         return super(DataFrame, self).append(other)
 
+    @derived_from(pd.DataFrame)
+    def iterrows(self):
+        for i in range(self.npartitions):
+            for row in self._default_get(self.dask, (self._name, i)).iterrows():
+                yield row
+
+    @derived_from(pd.DataFrame)
+    def itertuples(self):
+        for i in range(self.npartitions):
+            for row in self._default_get(self.dask, (self._name, i)).itertuples():
+                yield row
```

Comment:
Also, we may want to use the normal computation machinery.  This will allow people to use other schedulers if they desire (for example to avoid threads) and will send the graph through the appropriate optimizations.  

```python
for i in range(self.npartitions):
    df = self.get_division(i).compute()
    for row in df.itertuples():
        ...
```